build/common.sh

```diff
@@ -394,7 +394,7 @@ function kube::build::docker_build() {
   local -r context_dir=$2
   local -r pull="${3:-true}"
   local -r build_args=$4
-  local -ra build_cmd=("${DOCKER[@]}" build -t "${image}" "--pull=${pull}" "${build_args}" "${context_dir}")
+  local -ra build_cmd=("${DOCKER[@]}" build -t "${image}" "--pull=${pull}" ${build_args[@]} "${context_dir}")
```

Comment:
This should still be quoted.

Reply:
I've tried it quoted and it breaks. I think it's a glitch in how docker parses out parameters? If it's quoted, docker enforces it as a single parameter when there are actually multiple.

```
$ ./build/release.sh
+++ [0301 17:33:31] Verifying Prerequisites....
+++ [0301 17:33:35] Building Docker image kube-build:build-5f7bbf6510-5-v1.15.8-1
+++ Docker build command failed for kube-build:build-5f7bbf6510-5-v1.15.8-1

Sending build context to Docker daemon  8.704kB
Step 1/18 : ARG KUBE_BUILD_IMAGE_CROSS_TAG
Step 2/18 : ARG KUBE_BASE_IMAGE_REGISTRY
Step 3/18 : FROM ${KUBE_BASE_IMAGE_REGISTRY}/kube-cross:${KUBE_BUILD_IMAGE_CROSS_TAG}
invalid reference format

To retry manually, run:

docker build -t kube-build:build-5f7bbf6510-5-v1.15.8-1 --pull=false --build-arg=KUBE_BUILD_IMAGE_CROSS_TAG=v1.15.8-1 --build-arg=KUBE_BASE_IMAGE_REGISTRY=<redacted> /home/vagrant/kubernetes/_output/images/kube-build:build-5f7bbf6510-5-v1.15.8-1

!!! [0301 17:33:36] Call tree:
!!! [0301 17:33:36]  1: ./build/release.sh:35 kube::build::build_image(...)
```

Reply:
@BenTheElder Doing it with quotes actually runs the docker build command like
```
++ docker build -t kube-build:build-5f7bbf6510-5-v1.15.8-1 --pull=false '--build-arg=KUBE_BUILD_IMAGE_CROSS_TAG=v1.15.8-1 --build-arg=KUBE_BASE_IMAGE_REGISTRY=<redacted>/build-image' /home/vagrant/kubernetes/_output/images/kube-build:build-5f7bbf6510-5-v1.15.8-1
```

Reply:
I think I got it fixed for adding the quotes. Basically had to ensure `build_args` was received as a array, not just a string